payment/src/main/java/com/kyriba/conference/payment/domain/dto/Amount.java

```diff
@@ -12,6 +13,7 @@
 /**
  * @author Igor Lizura
  */
+@Getter
 @NoArgsConstructor
```

Comment:
I'd suggest to declare Amount as @Value and make it immutable, if possible. What do you think about it?